yaml/buddy-error-codes.yaml

```diff
@@ -286,6 +286,13 @@ Errors:
     id: "MB_HEATER_CONNECTED"
     approved: false
 
+  - code: "XX259"
+    printers: [COREONE, XL]
+    title: "FAILED TO REACH CHAMBER TEMPERATURE"
```

Comment:
IMHO the title is too long and won't fit, have you checked that?

Reply:
We currently don't display titles on the printer

Reply:
we do for the MMU errors, but otherwise ok